package.json

```diff
@@ -31,7 +31,9 @@
     "clipboard": "^2.0.0",
     "html5shiv": "^3.7.3",
     "jquery": "^1.12.4",
+    "metalsmith-rollup": "^1.0.0",
```

Comment:
Accidentally put dependencies in `devDependencies` and `dependencies` will resolve.